### ISSUE TITLE
View the BVDP Job Manager from within the project

### ISSUE THREAD

**Issue by bradtaylor** [DataBiosphere/terra-ui#10]
If a workflow submission includes failed jobs, the user needs to inspect these failures (viewing logs, etc) so that she can can debug the underlying error. Users also need to abort jobs that are running which may have been misconfigured, in order to avoid wasted spend.

Therefore, we will put a link to job details on each submission in the submissions history. This link will bring users to a filtered view of the Saturn Job Manager to accomplish these goals. 

[This](https://docs.google.com/document/d/1XYEzezW0WzlLOxVxJ3793gjbzOgEpMRnk3OV53jBFbo/edit) document describes options for Job Manager integration. Meeting about this document, the teams noted a strong desire for JM to have an independent presence within the application not related to Projects. Therefore, Job Manager will be presented as a discreet app in the Saturn umbrella with it's own domain. 

Significant open questions pertain to Job Manager integration:
* What instance of Job Manager will we be incorporating?
* What Cromwell is the JM instance pointed at?

The leading option discussed so far: FireCloud/Saturn submissions APIs and JM are both pointed to a fully secured Cromwell as a Service instance.

MAJOR DEPENDENCY: CaaS with SAM auth for FC/Saturn by mid September at the latest (ie enough lead time so we can integrate JM in time for October demo)

**Comment by bradtaylor** [comment]
See UI design mocks with a clickthrough path [here](https://invis.io/4XG2G91BPST)

**Comment by bradtaylor** [comment]
Needs visual design: How are we presenting the integration of JM with projects. Conversation needs Kylee's involvement

**Comment by dmohs** [comment]
Email from Kristian that had approval from most:
> Hi All -- I've been playing cat wrangler on this complex topic, and since it's impossible to get us all in a room I'm going to try and get us on the same page here.
>
> We have several incompatible realities:
> 1. We have a 3-month demo for the NIH, for which we want to use Saturn
> 2. Saturn will be launching jobs through the workspace service because
> 3. Cromwell as a Service will not be ready for Workbench use on this timeframe
> 4. Saturn has no way to monitor those jobs, and was depending on job manager which
> 5. Job Manager can only speak to CaaS, not the Cromwell inside FireCloud.
> 
> We've been through a number of possible technical solutions, all of which require work that is throwaway.  The major initial alternatives were
> 
> 1. Don't demo with Saturn @ the 3 month mark.  Not desirable b/c the NIH will only see the "final product" at the 6 month demo rather than incremental product.
> 
> 2. Rush CaaS into production for Workbench.  Extremely high risk, and not workable from a resource perspective
> 
> 3. Hack together the ability to have Job Manager talk to Rawls+Cromwell.  We struggled with this one the most, it's a bit of work distributed across several different teams, to make this happen (Job Manager, Saturn, Rawls, Devops) and will be thrown away when CaaS arrives.
> 
> Ultimately, the best path seems to be
> 
> 1. Add an extremely basic "view submission" tab to Saturn as temporary work.  It's all within one team, and they can crank on it pretty quickly.  Very, very, very basic... Like a list of the submissions and the overall status.
> 
> 2. For the demo, use Saturn launching through Rawls and viewing status in Saturn.
> 
> 3. We can show JM at the demo as a "not yet integrated, but here's how it works"
> 
> I get that it's not totally ideal, but I think it's the best comprise amongst all the constraints.

**Comment by seanhorgan** [comment]
This seems like a good compromise, especially for the 3 month demo.

**Comment by bradtaylor** [comment]
BLOCKED on availability of secure Cromwell as a Service at which both JM and Saturn submissions APIs can point

**Comment by seanhorgan** [comment]
When do we think the external dependencies will be unblocked so this work could move forward?

**Comment by bradtaylor** [comment]
This finally closed. It ended up being a duplicate of numerous other tickets beneath epic #1125
